### ISSUE TITLE
[Feature Request] Loader template change

### ISSUE THREAD

**Issue by leitosama** [cobbr/Covenant#134]
**Description**
Add a ability to change loader templates (like MSBuild and other). This feature can be helpful while generating and obfuscation non-binary loaders.

**Comment by cobbr** [comment]
Good idea @leitosama! Tagging this as Low Priority for now, so might take a bit to get to this, but would be happy to merge a PR.
